integration_tests/src/test/resources/compare/RunTimeoutIT-compare

```diff
@@ -65,16 +65,5 @@ RunTests
 Saved 1 shards to android_shards.json
   Uploading app-debug.apk \.*
   Uploading app-single-success-debug-androidTest.apk \.*
-  1 test / 1 shard
```

Comment:
Why those lines are removed?
I launched it locally and the timeout case failed because of it 🤔

Reply:
Locally I get the output without removed lines.

Reply:
I fixed this test by using ```test``` and ```apk``` from gcs. Using local files can produce different outputs related to file uploading speed.